src/lib/elements/table/cellCheck.svelte

```diff
@@ -24,7 +24,7 @@
     };
 </script>
 
-<TableCell class="u-position-relative">
+<TableCell>
     <div
         class="touch-area"
```

Comment:
This class is not needed anymore?

Reply:
this messes up the sticky table cells - I dont think there was ever a need for this class 🤔

Reply:
it was only needed because the class down below added position absoulute

Reply:
can we remove it then?